_posts/0000-01-02-ParvinBDJ.md

```diff
@@ -0,0 +1 @@
+--- layout: slide title: "Welcome to our second slide!" --- Your text Use the left arrow to go back!
```

Comment:
--- layout: slide title:
 "Welcome to our second slide!" 
--- Your text Use the left arrow to go back!
ajld _md_